src/mame/sega/segaic16_m.cpp

```diff
@@ -9,16 +9,13 @@
 #include "emu.h"
 #include "segaic16_m.h"
 
-#include <algorithm>
+#define LOG_DIVIDE  (1U << 1)
+#define LOG_COMPARE (1U << 2)
 
-//**************************************************************************
-//  DEBUGGING
-//**************************************************************************
-
-#define LOG_MULTIPLY    (0)
-#define LOG_DIVIDE      (0)
-#define LOG_COMPARE     (0)
+#define VERBOSE (0)
+#include "logmacro.h"
 
+#include <algorithm>
```

Comment:
`<algorithm>` should come before the logging stuff – logically it‘s more like inline code than a header as such.

Reply:
Moved it. This issue is likely present in other files as well.